server/src/main/java/io/druid/server/DruidNode.java

```diff
@@ -49,8 +48,26 @@
   private String host;
 
   @JsonProperty
-  @Min(0) @Max(0xffff)
-  private int port = -1;
+  @Max(0xffff)
```

Comment:
this one cannot have @ min(1024) validation now as we allow -1 value for port in case user sets `plaintextPort` which is the new property. Probably I can put @ deprecated annotation here.